shared/src/business/useCases/processStreamRecordsInteractor.js

```diff
@@ -154,7 +170,23 @@ exports.processStreamRecordsInteractor = async ({
       });
 
       if (failedRecords.length) {
-        for (const failedRecord of failedRecords) {
+        const catchAndReIndex = failedRecord => async e => {
+          await applicationContext
+            .getPersistenceGateway()
+            .createElasticsearchReindexRecord({
+              applicationContext,
+              recordPk: failedRecord.pk.S,
+              recordSk: failedRecord.sk.S,
+            });
+
+          applicationContext.logger.info(
```

Comment:
Same as above: can we use `logger.error`?

Reply:
Updated!